integreat_compass/cms/views/offers/my_offers_list_view.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+from django.contrib import messages
+from django.db import transaction
+from django.shortcuts import redirect, render
+from django.utils.decorators import method_decorator
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...decorators import permission_required
+from ...models import Offer
+
+logger = logging.getLogger(__name__)
+
+
+@method_decorator(permission_required("cms.change_offer"), name="dispatch")
+class MyOffersListView(TemplateView):
```

Comment:
```suggestion
class MyOffersListView(ListView):
```
I think it would make sense to let this inherit from Django's inbuilt [`ListView`](https://docs.djangoproject.com/en/4.2/ref/class-based-views/generic-display/#listview) and just override `get_queryset()` to filter the offers according to the current user.